src/components/views/settings/tabs/user/GeneralUserSettingsTab.tsx

```diff
@@ -351,9 +364,29 @@ export default class GeneralUserSettingsTab extends React.Component<IProps, ISta
             passwordChangeForm = null;
         }
 
+        let externalAccountManagement: JSX.Element;
+        if (this.state.externalAccountManagementUrl) {
+            const externalAccountManagementText = _t(
+                'You can manage your account <a>here</a>.',
```

Comment:
This is probably more a design comment, and not sure what the designs say, but I'd expect something more like `<a>Manage your account</a>`

Reply:
@SimonBrandner in terms of design review, it is this wording (and how it appears visually) that probably requires design review, please.